docs/pages/debugging/tools.mdx

```diff
@@ -46,7 +46,7 @@ The Developer menu provides the following options:
 - **Go Home**: To leave your app and navigate back to the dev client's or Expo Go app's Home screen.
 - **Toggle performance monitor**: To view the performance information about your app.
 - **Toggle element inspector**: To enable or disable the element inspector overlay.
-- **Open JS debugger**: To open Chrome DevTools which provides access to Console, Source, and Network tabs for apps using Hermes. For more information, see the [Debugging with Chrome DevTools](#debugging-with-chrome-devtools) section.
+- **Open JS debugger**: To open React Native DevTools which provides access to Console, Sources, Network, Memory, Components, and Profiler, tabs for apps using Hermes. For more information, see the [Debugging with React Native DevTools](#debugging-with-react-native-devtools) section.
```

Comment:
Small nit: React Native Dev Tools doesn't provide access to `Network` on it's own, that's still an Expo-only feature as it runs through our code.

Reply:
Thanks for the context! I've updated all the instances in this doc where we mention "Network" with added context that is (Expo only)

![CleanShot 2025-03-11 at 00 10 09](https://github.com/user-attachments/assets/c7182097-f26d-4883-a58f-a7b970ba12b8)

![CleanShot 2025-03-11 at 00 09 45](https://github.com/user-attachments/assets/4770113a-93e7-4c76-9fde-6d821d13a9ca)

![CleanShot 2025-03-11 at 00 14 44](https://github.com/user-attachments/assets/1f36037a-d40f-4b94-924a-e6049981204d)